### PR TITLE
Add alpinist submodule

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add alpinist submodule

## why
* Use `atlantis` deployment

### CODE REVIEW

**Comment by osterman** [comment]
atlantis/testing plan

**Comment by osterman** [comment]
atlantis/testing plan

**Comment by osterman** [comment]
atlantis/testing plan -w alpinist/terraform

**Comment by osterman** [comment]
atlantis/testing plan -p alpinist/terraform

**Comment by osterman** [comment]
atlantis/testing plan -p alpinist-terraform

**Comment by osterman** [comment]
atlantis/testing -p atlantis-repos plan

**Comment by osterman** [comment]
atlantis/testing -p atlantis-repos plan

**Comment by osterman** [comment]
atlantis/testing plan -p atlantis-repos

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[submodule "conf/alpinist"]
+	path = conf/alpinist
+	url = https://github.com/cloudposse/alpinist.git
+[submodule "conf/atlantis"]
+	path = conf/atlantis
+	url = https://github.com/cloudposse/geodesic-aws-atlantis.git
+	branch = init
```
Comment (aknysh) [inline .gitmodules]: `master` branch?

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ FROM cloudposse/terraform-root-modules:0.5.7 as terraform-root-modules
 
 FROM cloudposse/helmfiles:0.7.2 as helmfiles
```
Comment (aknysh) [inline Dockerfile:3]: `helmfiles` is at `0.8.0` now

File: conf/atlantis-repos/terraform.tfvars [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    extra_arguments "crud" {
+      commands = [
+        "apply",
+        "destroy",
+        "plan",
+      ]
+
+      arguments = [
+        "-lock-timeout=1m", 
+	"-no-color",
```
Comment (aknysh) [inline conf/atlantis-repos/terraform.tfvars:14]: format?

**Review by aknysh — COMMENTED** [review]
see comments

**Comment by joshmyers** [comment]
This is outdated by https://github.com/cloudposse/testing.cloudposse.co/pull/56

**Pull request closed without merging**